dstyle.dd

```diff
@@ -240,6 +240,31 @@ $(H3 $(LNAME2 properties, Properties))
         properties should be nouns, just like if they were member variables.
         Getter properties should $(I not) alter state.)
 
+$(H3 $(LNAME2 property-syntax, Property syntax))
+
+$(P Do not use $(LINK2 spec/function.html#pseudo-member, UFCS) outside of its intended use cases.
```

Comment:
I'm not sure why this is conflating UFCS with optional parens.

Reply:
I didn't see the spec had a section on optional parens, I thought that also fell under UFCS